modeldb/config.py

```diff
@@ -2,17 +2,15 @@
 
 import os
 
-# MDB_NEURON_MODELS_URL = "https://senselab.med.yale.edu/_site/webapi/object.json/?cl=19&oid=1882"
 MDB_NEURON_MODELS_URL = (
-    "http://modeldb.science/api/v1/models?modeling_application=NEURON"
-)
-MDB_MODEL_DOWNLOAD_URL = (
-    "https://senselab.med.yale.edu/_site/webapi/object.json/{model_id}"
+    "http://modeldb.science/api/v1/models/{model_field}?modeling_application=NEURON"
 )
+MDB_MODEL_METADATA_URL = "https://modeldb.science/api/v1/models/{model_id}"
+MDB_MODEL_DOWNLOAD_URL = "https://modeldb.science/eavBinDown?o={model_id}"
```

Comment:
Stylistic thing, no strong feelings, but instead of GET arguments, consider using:
```python
MDB_MODEL_DOWNLOAD_URL = "https://modeldb.science/download/{model_id}"
```
We continue to support the `eavBinDown` name from the old site, but we're no longer using an EAV architecture, so I'd rather not use the name.

Reply:
OK, this was just following the suggestion of https://github.com/neuronsimulator/nrn-modeldb-ci/issues/82#issue-1727804761

Reply:
Changed.

Reply:
Fair point. :)

Thanks